app/messaging/application/process-application.js

```diff
@@ -15,7 +15,31 @@ const processApplication = async (msg) => {
       throw new Error('Application validation error')
     }
 
-    const application = await getOrCreateApplication(applicationData)
+    const existingApplication = await applicationRepository.getBySbi(
```

Comment:
Can we confirm with business stakeholders and the product owner what the rule is here?

For example can we allow a single SBI to have one application per species or is the rule that one SBI can only have one successful application for a single species?

Reply:
Confirmed that the business rule is one application per SBI limited to a single species (for now).